app/views/home/_navbar.html.slim

```diff
@@ -18,6 +18,7 @@ nav.navbar.navbar-expand-lg.navbar-light.bg-light.my-zivi-top-navbar
         = navbar_link(t('home.navbar.links.organizations'), for_organizations_path)
         = navbar_link(t('home.navbar.links.about_us'), about_us_path)
         = navbar_link(t('home.navbar.links.blog'), blog_entries_path)
+        = navbar_link(t('home.navbar.links.civil_servant_faq'), zivi_faq_path)
```

Comment:
Maybe adjust path helper to `civil_servant_faq_path` so that is consistent with translation key and views